misc/selinux/cfengine-enterprise.fc

```diff
@@ -6,4 +6,4 @@
 /var/cfengine/bin/pg.*		    --	gen_context(system_u:object_r:cfengine_postgres_exec_t,s0)
 /var/cfengine/httpd/bin/.*      --	gen_context(system_u:object_r:cfengine_httpd_exec_t,s0)
 /var/cfengine/httpd/php/bin/.*  --	gen_context(system_u:object_r:cfengine_httpd_exec_t,s0)
-/opt/cfengine(/.*)?             --	gen_context(system_u:object_r:cfengine_var_lib_t,s0)
\ No newline at end of file
+/opt/cfengine(/.*)?             	gen_context(system_u:object_r:cfengine_var_lib_t,s0)
```

Comment:
Is it normal that these files should not have a newline at the end?

Reply:
I think it doesn't matter. So I'll make sure there's a newline at the end of the file in the follow-up work.